scripter/scripter.go

```diff
@@ -154,3 +167,75 @@ func SetBasicMethods(c ScrConn, service string) {
 		}
 	}, service)
 }
+
+// setScriptInterval sets the interval of checking whether scripts have been changed
+func SetScriptInterval(s Scripter) {
+
+	// How often to fire the passed in function
+	// in milliseconds
+	interval := 10 * time.Second
+
+	// Setup the ticket and the channel to signal
+	// the ending of the interval
+	ticker := time.NewTicker(interval)
+	quit := make(chan struct{})
+
+	// Put the selection in a go routine
+	// so that the for loop is none blocking
+	go func() {
+		for {
+			select {
+			case <- ticker.C:
+				go checkReloadScripts(s)
+			case <- quit:
+				ticker.Stop()
+				return
+			}
+
+		}
+	}()
+}
+
+// checkReloadScripts initializes services again when scripts have been changed within the service
+func checkReloadScripts(s Scripter) {
+	hasher := sha1.New()
+	for service, scripts := range s.GetScripts() {
+		isRenewService := false
+
+		// check for edited files
+		for _, script := range scripts {
+			content, err := ioutil.ReadFile(script.Source)
```

Comment:
What happens if the file contents are pretty big? Is this going to be slow to check the files every 10s?